src/Pay/Adapter/Stripe.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace Utopia\Pay\Adapter;
+
+use Utopia\Pay\Adapter;
+
+class Stripe extends Adapter {
+
+    private string $baseUrl = 'https://api.stripe.com/v1';
+    private string $secretKey;
+    private string $publishableKey;
+
+
+    public function __construct(string $publishableKey, string $secretKey, string $currency = 'USD')
```

Comment:
`publishableKey` is there a better name for this ? or is this from Stripe itself ?

Reply:
it's from stripe